test/extended/OWNERS

```diff
@@ -16,3 +17,4 @@ approvers:
   - ironcladlou
   - adambkaplan
   - danwinship
+  - coreydaley
```

Comment:
you shouldn't need this level of approval, add yourself to the test/extended/$subdirs your team owns tests or library code in.